core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/ScriptPostDto.java

```diff
@@ -24,6 +24,7 @@
     private Set<ScriptParameterDto> parameters;
     private Set<ActionDto> actions;
     private Set<ScriptLabelDto> labels;
+    private String deleted_At = "NA";
```

Comment:
this field should not be present in the ScriptPostDto.
When converting the ScriptPostDto to Script the deletedAt field of Script should be set to NA by default.

Also, create a public static final String NA_KEY = "NA " in Script and reference this field in every class in order to not make any typo's.